spec/helium/client/sensors_spec.rb

```diff
@@ -39,9 +43,13 @@
     expect(sensor.mac).to eq("6081f9fffe000478")
   end
 
+  it 'has a firmware version' do
+    expect(sensor.firmware).to eq("4020100")
+  end
+
   it 'has ports' do
     # TODO Port will be an object
-    expect(sensor.ports).to eq(["_se","lr","l","p","h","m","_e.info","t","b"])
+    expect(sensor.ports.sort).to eq(["_se","lr","l","p","h","m","_e.info","t","b"].sort)
```

Comment:
FYI, this `.sort` was added b/c I saw them come by in a different order on two different runs